monitor/flowgen/bridging/bridging.go

```diff
@@ -0,0 +1,90 @@
+package bridging
+
+import (
+	"context"
+	"time"
+
+	"github.com/omni-network/omni/contracts/bindings"
+	"github.com/omni-network/omni/e2e/app/eoa"
+	"github.com/omni-network/omni/lib/contracts/solvernet"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/ethclient/ethbackend"
+	"github.com/omni-network/omni/lib/evmchain"
+	"github.com/omni-network/omni/lib/log"
+	"github.com/omni-network/omni/lib/netconf"
+	"github.com/omni-network/omni/monitor/flowgen/types"
+	"github.com/omni-network/omni/monitor/flowgen/util"
+	"github.com/omni-network/omni/solver/app"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type job struct {
+	name    string
+	cadence time.Duration
+	network netconf.ID
+}
+
+func NewJob(network netconf.ID) types.Job {
+	return job{
+		name:    "Bridging",
+		cadence: 1 * time.Second,
+		network: network,
+	}
+}
+
+func (j job) Name() string {
+	return j.name
+}
+
+func (j job) Cadence() time.Duration {
+	return j.cadence
+}
+
+// Run bridges ETH from base to arbitrum.
+func (j job) Run(ctx context.Context, backends ethbackend.Backends) error {
+	token, found := app.AllTokens().Find(evmchain.IDMockL1, common.Address{})
+	if !found {
+		return errors.New("token not found")
+	}
+	deposit := app.Payment{Token: token, Amount: util.MilGwei}
+
+	expense, err := app.QuoteExpense(deposit.Token, deposit)
+	if err != nil {
+		return errors.Wrap(err, "quote expense")
+	}
+	user := eoa.MustAddress(j.network, eoa.RoleMonitor)
+
+	log.Info(ctx, "Running", "job", j.Name(), "deposit", deposit, "expense", expense, "user", user)
```

Comment:
Would be nice to filter on something like "flowgen" to see all flowgen logs.

So maybe: "Flowgen: running job", "name", j.Name, ...

Reply:
also avoid logging structs